src/js/pages/catalog/PackageDetailTab.js

```diff
@@ -103,9 +108,14 @@ class PackageDetailTab extends mixin(StoreMixin) {
   retrievePackageInfo() {
     const { packageName } = this.props.params;
     const { version } = this.props.location.query;
+    const cosmosPackages = CosmosPackagesStore.getPackages();
+    const selectedPackage = cosmosPackages.getPackageByName(packageName);
+
+    // fetch package versions available only if not cached
+    if (Object.keys(selectedPackage).length < 1) {
```

Comment:
Nit: `Object.keys(selectedPackage).length === 0` is clearer for checking empty object.

Reply:
This is minor to be a concern and block the PR because `Object.keys(selectedPackage).length < 1` is clear and not really different than `Object.keys(selectedPackage).length === 0`

Reply:
That's why I left as a comment and not a request 😄